theano/gpuarray/dnn.py

```diff
@@ -1626,11 +1630,12 @@ def apply(self, fgraph):
 
 @register_opt('cudnn', 'fast_compile')
 @op_lifter([SoftmaxGrad])
-def local_softmax_dnn_grad(node, ctx_name):
+#@register_opt2([SoftmaxGrad], 'fast_compile')
+def local_softmax_dnn_grad(op, ctx_name, inputs):
```

Comment:
When I add the tag, it would be using cudnn right ? Like how it registers as `fast_compile` when I add the tag ?